main.py

```diff
@@ -73,7 +73,32 @@
 
     await bot.process_commands(message)
 
+#start CuckTim Section
+#replaces "Cuck" with Rotating Light Emoji, meant to piss off Tim
+@bot.event
+async def on_message(message):
+    if not message.author.bot and "cuck" in message.content.lower():
+        CuckString = message.content.replace('cuck', ':rotating_light:')
```

Comment:
This line is unnecessary. The only reason the lamar/llama code has it is to maintain capitalization.